pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -298,9 +305,51 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) error {
 		return err
 	}
 
+	if err := o.getAPIBindings(ctx, newServerHost); err != nil {
+		// display the error, but don't stop the current workspace from being reported.
+		_, _ = fmt.Fprintf(o.Out, "error checking APIBindings: %v", err)
+	}
+
 	return currentWorkspace(o.Out, newServerHost, shortWorkspaceOutput(o.ShortWorkspaceOutput), workspaceType)
 }
 
+// getAPIBindings retrieves APIBindings within the workspace and look for ones without exported PermissionClaims.
+func (o UseWorkspaceOptions) getAPIBindings(ctx context.Context, host string) error {
+	_, clusterName, err := pluginhelpers.ParseClusterURL(host)
+	if err != nil {
+		return err
+	}
+
+	apiBindings, err := o.kcpClusterClient.Cluster(clusterName).ApisV1alpha1().APIBindings().List(ctx, metav1.ListOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return err
+	} else if apierrors.IsNotFound(err) {
+		// nothing to do if there aren't any
+		return nil
+	}
+
+	for _, binding := range apiBindings.Items {
```

Comment:
If you put this in its own function, you can unit test it more easily

Reply:
I saw this comment after I wrote the tests; I think that's still worth doing, as I'm not super happy with my tests as they stand.